fastchat/model/compression.py

```diff
@@ -226,3 +226,15 @@ def decompress(packed_data, config):
         return data[indices].contiguous()
     else:
         return data.view(original_shape)
+
+
+def load_gptq_model(model_name, load_gptq, wbits, groupsize, device, act_order=None):
+    from fastchat.modules import paths
+    from llama import load_quant
```

Comment:
btw, I'm not a contributor, but I couldn't compile the branch without

`sys.path.insert(0, os.path.abspath(Path("repositories/GPTQ-for-LLaMa")))
`

Reply:
Thanks for your feedback, I will check it later

Reply:
Sorry, I didn't reproduce the problem